lib/iterator.js

```diff
@@ -199,6 +199,66 @@ class Iterator {
     return result + suffix;
   }
 
+  // Find value in this iterator by comparing every value with
+  // the found one using `comparator`
+  // Signature: comparator[, accessor[, thisArg]]
+  //   comparator <Function> returns true if new value should be accepted
+  //     currValue <any> current value, starts with undefined
+  //     nextValue <any> next value
+  //     Returns: <boolean> true if next value should be accepted
```

Comment:
```suggestion
  //     Returns: <boolean> `true` if next value should be accepted
```
Ditto. And I'm also not a fan of description duplication.